addons/sourcemod/scripting/SaveLevel.sp

```diff
@@ -224,6 +224,9 @@ bool RestoreLevel(int client, const char[] sTarget, char[] sName = NULL_STRING,
 		}
 		else
 		{
+			if (client == -1)
```

Comment:
This check is already done here no?

https://github.com/srcdslab/sm-plugin-SaveLevel/pull/8/files#diff-d4d6d92f224f299071b68a2cfbd81633a32e0d21f720dbae4eeb6f009fa0b1a9R155

Reply:
Yes, but even with this we still have the error sometime.. https://github.com/srcdslab/sm-plugin-SaveLevel/issues/9
I tried a fix with this PR, but the error still here.

Reply:
maybe just this will do it:
- https://sourcemod.dev/#/entity/function.IsValidEntity

Reply:
Let me know what you think of this instead.

Reply:
we can give it a try